### PR TITLE
Improved the explanation about deployment + parameters.yml

### LINKED ISSUE #5158: Improve generic deployment article
From the article:

> **B) Configure your app/config/parameters.yml File**
> 
> This file should not be deployed, but managed through the automatic utilities provided by Symfony.

We should better explain what the "automatic utilities" are. Afaik, it's only the incenteev parameter handler that's included in the SE? Or is there something from Symfony itself that handles his.

The step is also a bit strange as it only tells what _not_ to do. Nothing explains what the user needs to do.

_This was issue was submitted by a user on a [dutch forum](http://www.phphulp.nl/php/forum/topic/symfony-vraag-mbt-deployen/97651/last/)_

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #5158.

As usual, the explanations in this article are a bit vague, because this is a generic deployment article and it contains no details (we have dedicated articles to explain those details).

### CODE REVIEW

**Comment by michaelperrin** [comment]
There is a little typo in your PR @javiereguiluz : change "insted" to "instead".

File: deployment.rst [hunk 1]
```diff
@@ -120,11 +120,18 @@ Check if your server meets the requirements by running:
...
+B) Configure your Parameters File
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Most Symfony applications define configuration parameters in a file called
+``app/config/parameters.yml``. This file should *not* be deployed, because
+Symfony generates it automatically using the ``app/config/parameters.yml.dist``
+file as a template (that's why ``parameters.yml.dist`` must be committed and
+deployed).
+
+If your application uses environment variables insted of these parameters, you
```
Comment (yceruto) [inline deployment.rst]: typo `insted` => `instead`

**Comment by javiereguiluz** [comment]
Fixed the typo. Thanks reviewers!

File: deployment.rst [hunk 2]
```diff
@@ -120,11 +120,18 @@ Check if your server meets the requirements by running:
 
     $ php app/check.php
 
-B) Configure your ``app/config/parameters.yml`` File
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-This file should *not* be deployed, but managed through the automatic utilities
-provided by Symfony.
+B) Configure your Parameters File
```
Comment (xabbuh) [inline deployment.rst]: we should add a label for the old headline

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**